stable/nginx-ingress/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+name: nginx-ingress
+version: 0.3.0
+description: An nginx Ingress controller that uses ConfigMap to store the nginx configuration.
+keywords:
```

Comment:
Can we add an icon here? e.g. pointing to https://cdn.wp.nginx.com/wp-content/themes/nginx-theme/assets/img/logo.png?